regexes/device/mobiles.yml

```diff
@@ -1267,9 +1267,17 @@ Rikomagic:
   device: 'tv'
   model: 'MK$1'
 
+# Senseit
+Senseit:
+   regex: 'SENSEIT[ _]?([^;/)]+) Build'
+   device: 'smartphone'
+   models:
+     - regex: '(R390|R450|E[54][10]0|A109|A200|L301)'
```

Comment:
Why are you defining a sub regex here? The global regex should already match everything in this case. You could directly use:

```
# Senseit
Senseit:
   regex: 'SENSEIT[ _]?([^;/)]+) Build'
   device: 'smartphone'
   model: '$1'
```

It can be changed as soon as some extra detections needs to be added